lib/matplotlib/patches.py

```diff
@@ -3710,23 +3727,25 @@ def __init__(self,
                      widthA=1., angleA=None,
                      widthB=1., angleB=None):
             """
-            *widthA*
-              width of the bracket
+            Parameters
+            ----------
+            widthA : float, optional, default : 1.0
+                Width of the bracket
 
-            *lengthA*
-              length of the bracket
+            lengthA : int
```

Comment:
I think you're right @dstansby .

@patniharshit can you update this docstring to more accurately reflect the call signature?